woocommerce/woocommerce-android
# Store creation: Fix site preview

This PR fixes the new store preview, which got "broken" by making the sites private by default after creation. Now the WebView must be authenticated before a preview can be shown.

**To test:**
1. Log in and navigate to the store picker
2. Tap on Add new site button and go through the flow (make sure you have free credits to complete this)
3. After a store is created and the preview screen is displayed, notice that the preview is rendered (takes pretty long)
